### PR TITLE
helm charts and Jenkinsfile

### PR DESCRIPTION
**Pull request by almerico**
For now full example took from old release it have to be new updatebot configuration to update helm with new version every time

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM openjdk:8-jdk-slim
```
Comment (ryandawsonuk) [inline Dockerfile]: @almerico Are we using this Dockerfile?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+     stage('Build Release') {
+       when {
+         branch 'master'
+       }
+       steps {
+         container('maven') {
+          dir('charts/activiti-cloud-acceptance-scenarios') {
+          //install helm chart for full example
+            sh "make install"
```
Comment (ryandawsonuk) [inline Jenkinsfile]: @almerico Wouldn't this fail if there's already a chart installed to that env (e.g. if pipeline is already deploying there)? Keep in mInd some of our service names are not prefixed with helm release name by default

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        container('maven') {
+          // sh "mvn versions:set -DnewVersion=$PREVIEW_VERSION"
+
+          sh "echo $PREVIEW_VERSION"
+          dir('charts/activiti-cloud-acceptance-scenarios') {
+          //install helm chart for full example
+            sh "make install" 
+          }
+          sh 'sleep 120'
+          sh "mvn clean install -DskipTests && mvn -pl '!apps-acceptance-tests,!multiple-runtime-acceptance-tests,!security-policies-acceptance-tests' clean verify"
```
[the remark this answers is not in the document]
Comment (almerico) [reply Jenkinsfile]: Preview works ok if run tests from Docker image for now I set up helm chart in separate namespace with full examples charts then run tests from Jenkins file against it.

File: Jenkinsfile (hunk 2 above)
Comment (almerico) [reply Jenkinsfile]: Namespace must be different for every pr

Comment (almerico) [reply Jenkinsfile]: It is configured in Make file in charts/ folder

**Pull request merged**